pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -166,10 +165,13 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication, ReentrancyGuard {
     constructor(
         address admin,
         IAuthentication vault,
```

Comment:
You can get the vault from the entrypoint, and it should always be the same one. Maybe we can remove this constructor argument?

Reply:
I think I can, and it actually makes some downstream stuff (deployer helpers) a bit cleaner. Will try.